DEVELOPMENT.md

```diff
@@ -96,4 +96,10 @@ docker ps -aq -f name=weave | xargs docker rm -f
 dontena cloud master rm
 ```
 
+You may also want to remove the cli config file to prevent previous state from causing issues:
+
+```bash
+rm $HOME/.kontena_client.json 
```

Comment:
Is `kontena master rm devmaster` not enough? Does it still leave some traces to config file?